### PR TITLE
improvements of contributing docs

### PR DESCRIPTION
**Pull request by orangesunny**
## Proposed changes

- rephrasing and adding information
- thanks @comradekingu for some inspiration
- slightly improved issue submission part

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [x] I have squashed my commits into logic units.
- [x] I have described the changes in the commit messages.

### CODE REVIEW

**Comment by orangesunny** [comment]
I will add some information to https://docs.weblate.org/en/latest/contributing/issues.html before merging this.

File: docs/contributing/index.rst [hunk 1]
```diff
@@ -45,15 +46,29 @@ Please respect these guidelines while writing:
...
 ---------------------------
 
-You can fund further Weblate development on the `donate page`_. Funds collected
-there are used to fund gratis hosting for libre software projects, and further
-development of Weblate. Please check the `donate page` for details, such
-as funding goals and rewards you can get for being a funder.
+You can boost Weblate’s development on the `donate page`_. Funds collected
+there are used to enable gratis hosting for libre software projects and further
+development of Weblate.
+Please check the `donate page` for options, such
```
Comment (nijel) [inline docs/contributing/index.rst:70]: Link missing: 
```suggestion
Please check the `donate page`_ for options, such
```

**Comment by comradekingu** [comment]
This isn't documentation. It is worse. It has errors, and it is poorly written.
It also ruins the style used in the entire documentation, which I have spent time going to twice as a whole, and used most of my time editing other than that. I took pride in that.
https://github.com/WeblateOrg/weblate/pull/5566#issuecomment-801842143
Here you have an example of the disconnect between being asked how to do something, and the idea that it makes one capable of answering.

**Comment by orangesunny** [comment]
> This isn't documentation. It is worse. It has errors, and it is poorly written.
> I don't see why I should try to make it look better, again. I did so the first time for why I would take different lessons from it if someone took the time to do it to me.
> 
> It also ruins the style used in the entire documentation, which I have spent time going through twice as a whole, and used most of my time editing other than that. I took pride in that.


It is a guide. It respects the original style of this article. If you feel it’s wrong, you can suggest edits, but deleting much of it is not a way to go. Also, you don’t have to agree with this part of the writing, leave it as is, and aim your efforts on some other parts.

**Comment by orangesunny** [comment]
@nijel can you please check the failing checks? I feel blind...

File: docs/contributing/index.rst [hunk 2]
```diff
@@ -45,15 +46,29 @@ Please respect these guidelines while writing:
...
+
+    You can `translate the docs <https://hosted.weblate.org/projects/weblate/documentation/>`_.
+
+
+.. _discussions:
+
+Weblate discussions
+-------------------
+
+If you have an idea and you are not sure if it’s suitable for an issue, don’t worry. You can join the community in the `discussions <https://github.com/WeblateOrg/weblate/discussions>`_.
```
Comment (nijel) [inline docs/contributing/index.rst:59]: This is the problem - the label here matches the one explicitly defined above:
```suggestion
If you have an idea and you are not sure if it’s suitable for an issue, don’t worry. You can join the community in the `discussions <https://github.com/WeblateOrg/weblate/discussions>`_.
```
```suggestion
If you have an idea and you are not sure if it’s suitable for an issue, don’t worry. You can join the community in the `discussions <https://github.com/WeblateOrg/weblate/discussions>`_.
```

```suggestion
If you have an idea and you are not sure if it’s suitable for an issue, don’t worry. You can join the community in the `GitHub discussions <https://github.com/WeblateOrg/weblate/discussions>`_.
```

**Comment by orangesunny** [comment]
@comradekingu Added one more part from #5688 but rephrased it. Please check and tell.

**Comment by comradekingu** [comment]
@orangesunny If your directions are anything to go by, this should sort itself out.
Feel free to use my descriptions of this document in referencing its quality.
People looking for stuff to fix need look no further.

**Pull request merged**